recipes/placekey/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "placekey" %}
+{% set version = "0.0.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 25235e7d5733ba229464fa4de0dacfeda84063b9fbb52041ac67d04c9c35d6be
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - backoff
```

Comment:
are all of these deps required at build time?

Reply:
Hi @mariusvniekerk 
Honestly I am not sure what should be required at build time. I just mentioned dependencies here.
Would you kindly let me know how to check if a specific package is required at build time?

Thanks,
Ming

Reply:
Odds are you just need python and pip, i'd recommend removing all of them.  If you use a tool like [greyskull](https://github.com/conda-incubator/grayskull) to generate the recipe that will generally do this for you.

Reply:
Okay, thanks. Let me try with greyskull.